test/e2e/push_test.go

```diff
@@ -141,6 +141,44 @@ var _ = Describe("Podman push", func() {
 		Expect(output).To(ContainSubstring("zstd"))
 	})
 
+	It("push test --force-compression --format=v2s2", func() {
+		if podmanTest.Host.Arch == "ppc64le" {
+			Skip("No registry image for ppc64le")
+		}
+		if isRootless() {
+			err := podmanTest.RestoreArtifact(REGISTRY_IMAGE)
+			Expect(err).ToNot(HaveOccurred())
+		}
+		lock := GetPortLock("5005")
+		defer lock.Unlock()
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "registry", "-p", "5005:5000", REGISTRY_IMAGE, "/entrypoint.sh", "/etc/docker/registry/config.yml"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+
+		if !WaitContainerReady(podmanTest, "registry", "listening on", 20, 1) {
+			Skip("Cannot start docker registry.")
+		}
+
+		session = podmanTest.Podman([]string{"build", "-t", "imageone", "build/basicalpine"})
```

Comment:
@Luap99 Do we need to (or, do for the future, do we want to) worry about running the tests concurrently? 

AFAICS they would clash in this image tag, as well as the `registry` container name.

Reply:
Everything in test/e2e is already running concurrently (with very few exceptions, look for Serial decorator). The way e2e tests are setup is that each test case has its own storage (--root/--runroot/--tmpdir) so there are never any conflicts with names so this should be fine here.

Reply:
Isn't that why the registry for each test is starter on a different port?